CTSK.py

```diff
@@ -0,0 +1,280 @@
+
+    def __init__(self, sequence: str, alphabet: dict = _alphabet,
+                 complement_alphabet: dict = _complement_alphabet):
+        super().setter(sequence, alphabet, complement_alphabet)
+
+
+class AminoAcidSequence(BiologicalSequence):
+    alphabet = {'V', 'I', 'L', 'E', 'Q', 'D', 'N', 'H', 'W', 'F', 'Y', 'R', 'K', 'S', 'T', 'M', 'A', 'G', 'P',
+                'C', 'v', 'i', 'l', 'e', 'q', 'd', 'n', 'h', 'w', 'f', 'y', 'r', 'k', 's', 't', 'm', 'a', 'g', 'p', 'c'}
+    average_weights = {
+        'A': 71.0788, 'R': 156.1875, 'N': 114.1038, 'D': 115.0886, 'C': 103.1388,
+        'E': 129.1155, 'Q': 128.1307, 'G': 57.0519, 'H': 137.1411, 'I': 113.1594,
+        'L': 113.1594, 'K': 128.1741, 'M': 131.1926, 'F': 147.1766, 'P': 97.1167,
+        'S': 87.0782, 'T': 101.1051, 'W': 186.2132, 'Y': 163.1760, 'V': 99.1326
+    }
+    three_letter_alphabet = {'A': 'Ala', 'C': 'Cys', 'D': 'Asp', 'E': 'Glu', 'F': 'Phe', 'G': 'Gly', 'H': 'His',
+                             'I': 'Ile', 'K': 'Lys', 'L': 'Leu',
+                             'M': 'Met', 'N': 'Asn', 'P': 'Pro', 'Q': 'Gln', 'R': 'Arg', 'S': 'Ser', 'T': 'Thr',
+                             'V': 'Val', 'W': 'Trp', 'Y': 'Tyr'}
+
+    def __init__(self, sequence: str):
+        self.sequence = sequence
+
+    def __str__(self):
+        return (str(self.sequence))
+    
+    
+    def check_alphabet(self) -> bool:
+        """
+        Check if a sequence is DNA or RNA.
+
+        Returns:
+            bool: True if the sequence is DNA or RNA, False otherwise.
+        """
+
+        unique_chars = set(self.sequence)
+        if not (unique_chars <= self.alphabet):
+            return False
+        return True  
+    
+    
+    def aa_average_weight(self, weight: str = 'average') -> float:
+        """
+        Calculate the amino acids weight in a protein sequence.
+
+        Args:
+            seq (str): The amino acid sequence to calculate the weight for.
+            weight (str, optional): The type of weight to use, either 'average' or 'monoisotopic'. Default is 'average'.
+
+        Returns:
+            float: The calculated weight of the amino acid sequence.
+        """
+
+        final_weight = 0
+        for aa in self.sequence.upper():
+            final_weight += self.average_weights[aa]
+        return round(final_weight, 3)
+
+
+    def one_to_three_letter_code(self) -> str:
+        """
+        This function converts a protein sequence from one-letter amino acid code to three-letter code.
+
+        Args:
+            sequence (str): The input protein sequence in one-letter code.
+
+        Returns:
+            str: The converted protein sequence in three-letter code.
+        """
+
+        three_letter_code = [self.three_letter_alphabet.get(aa.upper()) for aa in self.sequence]
+        return '-'.join(three_letter_code)
+
+
+def filter_dna(input_path: str, output_filename: str = '', gc_bounds: Union[Tuple[int, int], int] = (0, 100),
```

Comment:
```suggestion
def filter_fastq(input_path: str, output_filename: str = '', gc_bounds: Union[Tuple[int, int], int] = (0, 100),
```
минутка душноты... мы же записи fastq фильтруем. Ну и еще filter_dna звучит забавно))